app/views/ama_layout/agent/_siteheader.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="header">
+  <div class="title-bar" data-responsive-toggle="main-menu" data-hide-for="large">
+    <button class="menu-icon" type="button" data-toggle="offCanvasLeft"></button>
+    <%= link_to(image_tag("ama-logo.png"), "https://www.ama.ab.ca", class: "title-bar__logo") %>
```

Comment:
Is there a config variable for the WWW URL you can use? I see the other layout hard-codes it to prod too anyways.